src/k8s-engine/lima.ts

```diff
@@ -1433,6 +1434,28 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
     await this.ssh('sudo', 'mv', './trivy', '/usr/local/bin/trivy');
   }
 
+  protected async installGuestAgent(kubeVersion: semver.SemVer | null, debug = false) {
+    const guestAgentPath = path.join(paths.resources, 'linux', 'internal', 'rancher-desktop-guestagent');
+
+    await Promise.all([
+      (async() => {
+        await this.lima('copy', guestAgentPath, `${ MACHINE_NAME }:./rancher-desktop-guestagent`);
+        await this.ssh('sudo', 'mv', './rancher-desktop-guestagent', '/usr/local/bin/rancher-desktop-guestagent');
+      })(),
+      this.writeFile('/etc/init.d/rancher-desktop-guestagent', SERVICE_GUEST_AGENT_INIT, 0o755),
+      (async() => {
+        const kube = K3sHelper.requiresPortForwarding(kubeVersion);
+
+        await this.writeConf('rancher-desktop-guestagent', {
+          GUESTAGENT_KUBERNETES: kube ? 'true' : 'false',
+          GUESTAGENT_IPTABLES:   'false',
+          GUESTAGENT_DEBUG:      debug ? 'true' : 'false',
```

Comment:
I'm guessing that when `GUESTAGENT_DEBUG` is `true`, the guest agent prints extra logs? If so, can we use `LimaBackend.debug` here? That way it ties into our application-wide debug mode setting.

Reply:
Err, yes, I totally forgot that existed :D Fixed in 480ff4322e6a1fbfb663eca7fd5f7152c8b90c35